src/gwt/src/org/rstudio/studio/client/workbench/views/source/editors/text/TextEditingTarget.java

```diff
@@ -2179,6 +2181,11 @@ public TextFileType getTextFileType()
    }
    
    @Handler
+   void onShowDocumentOutline()
+   {
```

Comment:
Ah, yes, we do. Currently the TextEditingTarget doesn't 'know' about the associated document outline and so I am not sure if the handler should live here. Any thoughts? We could:

- construct the `TextEditingTarget` with a reference to the outline widget,
- attach the handler to the widget itself (or something else a bit higher up),
- use events to coordinate

Any thoughts one what might fit best here?